tools/ci/inria/extra-checks

```diff
@@ -21,7 +21,8 @@
 # In this context, it is necessary to skip a few tests whose behaviour
 # is modified by the instrumentation:
 
-export OCAMLTEST_SKIP_TESTS="tests/afl-instrumentation/afltest.ml tests/runtime-errors/stackoverflow.ml"
+export OCAMLTEST_SKIP_TESTS="tests/afl-instrumentation/afltest.ml "\
```

Comment:
This looks a bit strange. Why not just split the line with a backslash inside the quotes?